gammapy/utils/scripts.py

```diff
@@ -185,25 +185,38 @@ def make_path(path):
     return Path(expandvars(str(path)))
 
 
-def recursive_update_dict(old, new):
-    """Recursively update a dict with new values
+def recursive_merge_dicts(a, b):
+    """Recursively merge two dictionaries, entries in b override entries in a.
```

Comment:
Keep the first sentence short so that it fits in the automodapi summary tables. In this case I'd suggest:
```
"""Recursively merge two dictionaries.

Entries in b override entries in a.
...
```